src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -93,69 +93,80 @@ public InventoryItemManager(RestClient restClient,
   }
 
 
-  /**
-   * Returns list of item records for specified id's.
-   *
-   * @param ids List of item id's
-   * @return future with list of item records
-   */
```

Comment:
why java docs deleted, you you want to extract interfaces and move java docs to interfaces?